libvast/test/format/json.cpp

```diff
@@ -118,23 +118,96 @@ TEST(json to data) {
   CHECK_EQUAL(materialize(slice.at(0, 17)), data{reference});
 }
 
-#if 0
-TEST(json suricata) {
-  using reader_type = format::json::reader<format::json::suricata_selector>;
-  auto input = std::make_unique<std::istringstream>(std::string{eve_log});
-  reader_type reader{caf::settings{}, std::move(input)};
+TEST(json reader - suricata) {
+  using factory = vast::factory<format::reader>;
+  factory::initialize();
+  auto reader = factory::get("suricata")(caf::settings{});
```

Comment:
The idiomatic way to use this would be `factory<format::reader>::make(...)` to directly create the reader.